### ISSUE TITLE
[New rule] abstract_field

### ISSUE THREAD

**Issue by fzyzcjy** [dart-code-checker/dart-code-metrics#1075]
**Please describe what the rule should do:**

```dart
abstract class A {
  double get a;
  set a(double value);
}
```

can be written as following:

```dart
abstract class A {
  abstract double a;
}
```

The linter should warn for the first case (so users can write the second case)

**If your rule is inspired by other please provide link to it:**

**What category of rule is this? (place an "X" next to just one item)**

- [ ] Warns about a potential error (problem)
- [x] Suggests an alternate way of doing something (suggestion)
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

```dart

```

**Are you willing to submit a pull request to implement this rule?**
